src/page/Howto/Recover_from_a_deleted_admin_user.rst

```diff
@@ -0,0 +1,170 @@
+Recover_from_a_deleted_admin_user
```

Comment:
You don't need the underscores in the title. Most pages have them just because of the automation used to insert them.